docs/auditors/mounts.md

```diff
@@ -0,0 +1,273 @@
+| :------ | :-------- | :------------------------------------------------------------------- | :----------------------------------------------------------------------- |
+| -s      | --paths   | List of sensitive paths that shouldn't be mounted.                   | [default sensitive host paths list](#Default-sensitive-host-paths-list)  |
+
+Also see [Global Flags](/README.md#global-flags)
+
+#### Default sensitive host paths list
+
+| Host path                 | Description                                                                              |
+| :------------------------ | :--------------------------------------------------------------------------------------- |
+| /proc                     |  Pseudo-filesystem which provides an interface to kernel data structures                                                  |
+| /var/run/docker.sock      |  Unix socket used to communicate with Docker daemon                                   |
+| /                         |  Filesystem's root |
+| /etc                      |  Directory that usually contains all system related configurations files         |
+| /root                     |  Home directory of the `root` user                                             |
+| /var/run/crio/crio.sock   |  Unix socket used to communicate with the CRI-O Container Engine                                                 |
+| /home/admin               |  Home directory of the `admin` user        |
+| /var/lib/kubelet          |  Directory for Kublet-related configuration                                                             |
+| /var/lib/kubelet/pki      |  Directory containing the certificate and private key of the Kublet                                                               |
+| /etc/kubernetes           |  Directory containing Kubernetes related configuration              |
+| /etc/kubernetes/manifests |  Directory containing manifest of Kubernetes components                                                         |
+
+## Examples
+
+```
+$ kubeaudit mounts -f auditors/mounts/fixtures/proc-mounted.yml
+
+---------------- Results for ---------------
+
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: pod
+    namespace: proc-mounted
+
+--------------------------------------------
+
+-- [error] SensitivePathsMounted
+   Message: Sensitive path mounted as volume: proc-volume (/proc -> /host/proc, readOnly: false). It should be removed from the container's mounts list.
+   Metadata:
+      Container: container
+      MountName: proc-volume
+      MountPath: /host/proc
+      MountReadOnly: false
+      MountVolume: proc-volume
+      MountVolumeHostPath: /proc
+
+```
+
+### Example with Config File
+
+A custom paths list can be provided in the config file. See [docs](docs/all.md) for more information. These are the host
+paths you'd like to have kubeaudit raise an error when they are mounted in a container.
+
```

Comment:
```suggestion
If you don't want kubeaudit to raise errors for all the paths in the default list (`DefaultSensitivePaths`), you can provide a custom paths list in the config file. See [docs](docs/all.md) for more information. That way kubeaudit will only raise errors for those specific paths listed in the config file.
```

Reply:
something along these lines... Because what happens here is that it overrides the default list, so it will only wtach the paths I specify